docs/cpp/compiler-limits.md

```diff
@@ -11,9 +11,9 @@ The C++ standard recommends limits for various language constructs. The followin
 
 - Nesting levels of compound statements, iteration control structures, and selection control structures - C++ standard: 256, Microsoft C++ compiler: depends on the combination of statements that are nested, but generally between 100 and 110.
 
-- Parameters in one macro definition - C++ standard: 256, Microsoft C++ compiler: 127.
+- Parameters in one macro definition - C++ standard: 256, Microsoft C++ compiler: 32767.
```

Comment:
`/Zc:preprocessor-` is still the default for most compiler modes so this may need to be qualified as "under `/Zc:preprocessor-`:127, under `/ZC:preprocssor`:32767"

Reply:
@anporumb  - Rastaban's comment seems like an important qualification. I'll hold off merging this until that's added.